src/lib/I18n.js

```diff
@@ -64,17 +78,17 @@ class I18n {
      * @param {string} language - ISO 639-1 language codes, e.g. en, en-us, de, de-at
      * @returns {string}
      */
-    getClosestSupportedLanguage(language) {
+    static getClosestSupportedLanguage(language) {
         if (language in this._dict) return language;
         language = language.split('-')[0];
         if (language in this._dict) return language;
         const languagePrefix = `${language}-`;
         for (const supportedLanguage of this.availableLanguages()) {
             if (supportedLanguage.startsWith(languagePrefix)) return supportedLanguage;
         }
-        // language is not supported. Return a variant of 'en' if available, otherwise an arbitrary supported language
-        if (language !== 'en') {
-            return this.getClosestSupportedLanguage('en');
+        // language is not supported. Return a variant of this._fallbackLanguage if available, otherwise an arbitrary supported language
+        if (language !== this._fallbackLanguage) {
+            return this.getClosestSupportedLanguage(this.fallbackLanguage);
```

Comment:
will end up as endless recursion if fallback language has a hyphen, e.g. 'en-us' or if _fallbackLanguage is undefined